weblogo/logo.py

```diff
@@ -626,13 +632,15 @@ def __init__(self, logodata, logooptions=None):
 # End class LogoFormat
 
 
-def parse_prior(composition, alphabet, weight=None):
+def parse_prior(
+    composition: Any, alphabet: Alphabet, weight: float = None
+) -> Optional[np.ndarray]:
```

Comment:
@gecrooks I typed `composition: Any` b/c it seemed like composition accepted dictionary, float, or string, or None. 
But looking at this now, would `Union[str, dict, None]` also work?